.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
+npm test
```

Comment:
Must have: Let's remove `npm test` as the original intention of the ticket is to only run an automatic prettier fix.

Reply:
Done.